pkg/controller/controller.go

```diff
@@ -198,28 +198,28 @@ func (c *Controller) findAllClusters() (string, error) {
 }
 
 func (c *Controller) initResource() (string, error) {
+	watchVersion := "0"
```

Comment:
This is needed by this PR.

Here's what this change is: Previously, we only checks TPR and if TPR exists then we ignores all other resources. With this change, we want to make sure we checks each resource and ignores only if that resource exists.

It's necessary in this PR. Because we have supported different storage classes.